tests/functional_tests/models/deepseek/test_deepseek_conversion.py

```diff
@@ -56,7 +58,10 @@ def deepseek_toy_model_path(self, tmp_path_factory):
         model_dir = temp_dir / "deepseek_toy"
 
         # Create a minimal config and model using Auto classes to avoid direct imports
-        config = AutoConfig.from_pretrained("deepseek-ai/DeepSeek-V3", trust_remote_code=True)
+        config = AutoConfig.from_pretrained(
+            "deepseek-ai/DeepSeek-V3",
+            trust_remote_code=is_safe_repo(hf_path="deepseek-ai/DeepSeek-V3"),
```

Comment:
i dont get how this is an improvement for security..could you explain why its needed here?